packages/expo-camera/android/src/main/java/expo/modules/camera/CameraViewModule.kt

```diff
@@ -228,6 +228,9 @@ class CameraViewModule : Module() {
       }
 
       Prop("barCodeScannerSettings") { view: ExpoCameraView, settings: Map<String, Any?>? ->
+        if (settings == null) {
+          return@Prop
```

Comment:
Shouldn't we reset these settings?

Reply:
We can't. It will be reset automatically when someone disables the barcode.